test/e2e/egressip.go

```diff
@@ -661,7 +662,7 @@ var _ = ginkgo.DescribeTableSubtree("e2e egress IP validation", feature.EgressIP
 			if err != nil {
 				return false, err
 			}
-			ips, err := util.DefaultNetworkPodIPs(pod)
+			ips, err := podannotation.DefaultNetworkPodIPs(pod)
```

Comment:
i think in E2E we are trying to avoid using OVNK functional code

Reply:
then i am lucky that it is not a part of this PR anymore :P